pkg/karmadactl/register/register.go

```diff
@@ -187,6 +187,7 @@ func NewCmdRegister(parentCommand string) *cobra.Command {
 	flags.StringVar(&opts.ClusterRegion, "cluster-region", "", "The region of the joining cluster. The Karmada scheduler can use this information to spread workloads across regions for higher availability.")
 	flags.StringSliceVar(&opts.ClusterZones, "cluster-zones", []string{}, "The zones of the joining cluster. The Karmada scheduler can use this information to spread workloads across zones for higher availability.")
 	flags.BoolVar(&opts.EnableCertRotation, "enable-cert-rotation", false, "Enable means controller would rotate certificate for karmada-agent when the certificate is about to expire.")
+	flags.StringVar(&opts.KarmadaDir, "karmada-dir", KarmadaDir, "The directory Karmada owns for storing various configuration files.")
```

Comment:
```suggestion
	flags.StringVar(&opts.KarmadaConfigDir, "karmada-config-dir", KarmadaDir, "The directory Karmada owns for storing various configuration files. Defaults to /etc/karmada.")
```

Reply:
I've checked the `karmadactl register` command and noticed that there is a `--karmada-data` flag available. What do you think about using this flag name for consistency?

```bash
$ karmadactl register --help
...
--karmada-data='/etc/karmada':
Karmada data path. Contains kubeconfig, certificates, and CRDs files.
...
```

Reply:
> What do you think about using this flag name for consistency?

nice suggestion